spec/controllers/admin/users_controller_spec.rb

```diff
@@ -19,4 +19,24 @@
     end
   end
 
+  describe '#lock_account' do
+    it 'queues a job to lock the given account' do
```

Comment:
Uh, I should have payed closer attention to this earlier, sorry. This doesn't reflect what the method does and neither what the spec tests for. It does not queue a job, it executes that action directly (which is fine, since it's a single `UPDATE` query). So please update the wording here, something like "it locks the given account" should be fine.

Remember that you can edit your commit with `git commit --amend`.